rancid-tomatillos/src/App.js

```diff
@@ -3,27 +3,43 @@ import React, {Component} from "react";
 import movieData from "./sample-data/movie-data.js"
 import Navbar from "./Components/Navbar/Navbar"
 import CardContainer from "./Components/CardContainer/CardContainer"
+import MovieView from "./Components/MovieView/MovieView"
 
 class App extends Component {
   constructor() {
     super();
     this.state= {
       allMovies: movieData.movies,
+      isClicked: false,
+      selectedMovie: {}
+      }
     };
-  }
+  
+
+   handleClick = (id) => {
```

Comment:
Nice work handling your logic here.  Curious if it's necessary to set the whole movie to state or if you could just set the id to state?